Linear_cell_complex/include/CGAL/Linear_cell_complex_constructors.h

```diff
@@ -339,9 +339,9 @@ namespace CGAL {
           else if ( it->vertex(1) == atr.infinite_vertex() )
             dart = alcc.beta(res, 1);
           else if ( it->vertex(2) == atr.infinite_vertex() )
-            dart = alcc.beta(res, 2);
+            dart = alcc.beta(res, 1, 1);
```

Comment:
beta(res, 2) and beta(res, 1, 1) are two darts belonging to the same vertex (cf the figure http://doc.cgal.org/latest/Linear_cell_complex/classCGAL_1_1Linear__cell__complex.html#a9d90420e64f48ff94620ca062310a13a)

Thus your modification does not modify anything for the infinite vertex